### DISCUSSION TITLE
Is relay supported by sending domain?

### DISCUSSION THREAD

**Discussion by SKJoy** [Mailu/Mailu#3174] [User support]
**This is not a issue/bug report. Just didn't know where to ask about it.**

# **Question**: Is there a way to relay SMTP through external SMTP servers depending on the sending domain hosted with MailU?

### **Confusion**: I have seen there is a domain based relay section in the management panel. But looks like that works depending on the target domain, like use an external SMTP relay server if the target email address belongs to GMAIL.COM or something like that. While I want this to happen depending on the sender domain.

**Thanks** in advance and needless to say, **MailU** is just `GREAT` :)

**Comment by medram** [comment]
I'm facing the same issue, have you found any solution?

**Reply by SKJoy** [reply]
Not yet :( And looks like this is a very solitary place!

**Comment by medram** [comment]
Anyway, just ensure to run this command after updating the mailu.env file to take effect:
```bash
docker-compose up -d
```
